src/pages/ShareCodePage.tsx

```diff
@@ -0,0 +1,129 @@
+                    .join(' & ');
+            }
+
+            return ReportUtils.getParentNavigationSubtitle(report).workspaceName ?? ReportUtils.getChatRoomSubtitle(report);
+        }
+
+        return session?.email;
+    };
+
+    const title = isReport ? ReportUtils.getReportName(report) : currentUserPersonalDetails.displayName ?? '';
+    const subtitle = getSubtitle();
+    const urlWithTrailingSlash = Url.addTrailingForwardSlash(environmentURL);
+    const url = isReport ? `${urlWithTrailingSlash}${ROUTES.REPORT_WITH_ID.getRoute(report.reportID)}` : `${urlWithTrailingSlash}${ROUTES.PROFILE.getRoute(session?.accountID ?? '')}`;
+    const platform = getPlatform();
+    const isNative = platform === CONST.PLATFORM.IOS || platform === CONST.PLATFORM.ANDROID;
+
+    return (
+        <ScreenWrapper
+            testID={ShareCodePage.displayName}
+            shouldShowOfflineIndicatorInWideScreen={!isReport}
+        >
+            <HeaderWithBackButton
+                title={translate('common.shareCode')}
+                onBackButtonPress={() => Navigation.goBack(isReport ? ROUTES.REPORT_WITH_ID_DETAILS.getRoute(report.reportID) : ROUTES.SETTINGS)}
+                shouldShowBackButton={isReport || isSmallScreenWidth}
+            />
+            <ScrollView style={[themeStyles.flex1, themeStyles.mt3]}>
+                <View style={[isSmallScreenWidth ? themeStyles.workspaceSectionMobile : themeStyles.workspaceSection, themeStyles.ph4]}>
+                    <QRShareWithDownload
+                        ref={qrCodeRef}
+                        url={url}
+                        title={title}
+                        subtitle={subtitle}
+                        logo={isReport ? expensifyLogo : (UserUtils.getAvatarUrl(currentUserPersonalDetails?.avatar, currentUserPersonalDetails?.accountID) as ImageSourcePropType)}
+                        logoRatio={isReport ? CONST.QR.EXPENSIFY_LOGO_SIZE_RATIO : CONST.QR.DEFAULT_LOGO_SIZE_RATIO}
+                        logoMarginRatio={isReport ? CONST.QR.EXPENSIFY_LOGO_MARGIN_RATIO : CONST.QR.DEFAULT_LOGO_MARGIN_RATIO}
+                    />
+                </View>
+
+                <View style={{marginTop: 36}}>
+                    <ContextMenuItem
+                        isAnonymousAction
+                        text={translate('qrCodes.copy')}
+                        icon={Expensicons.Copy}
+                        successIcon={Expensicons.Checkmark}
+                        successText={translate('qrCodes.copied')}
+                        onPress={() => Clipboard.setString(url)}
+                        shouldLimitWidth={false}
+                    />
+
+                    {isNative && (
+                        <MenuItem
+                            isAnonymousAction
+                            title={translate('common.download')}
+                            icon={Expensicons.Download}
+                            onPress={qrCodeRef.current?.download}
```

Comment:
```suggestion
                            onPress={() => qrCodeRef.current?.download?.()}
```
This caused regression. Now deploy blocker - https://github.com/Expensify/App/issues/35919

@filip-solecki are you around to fix quickly?

Reply:
Yes, will fix in a moment

Reply:
@situchan - fixed [here](https://github.com/Expensify/App/pull/35924)